packages/mosip-api/src/routes/event-review.ts

```diff
@@ -110,14 +110,14 @@ export const reviewEventHandler = async (
     informantType !== "FATHER" &&
     informantType !== "SPOUSE"
   ) {
-    let relatedPerson = findEntry(
+    let relatedPerson = findEntry<fhir3.RelatedPerson>(
       "informant-details",
       composition,
       request.body,
-    ) as fhir3.RelatedPerson;
+    );
     const informant = getFromBundleById(
       request.body,
-      relatedPerson.patient.reference!.split("/")[1],
+      relatedPerson?.patient.reference!.split("/")[1] ?? "throws-to-catch",
```

Comment:
`getFromBundleById` will throw if the id is not found. This now defaults to an id that is not definitely in the bundle: `throws-to-catch`.